### ISSUE TITLE
Can't resolve MuiThemeProvider

### ISSUE THREAD

**Issue by marcpiechura** [cmeeren/Feliz.MaterialUI#27]
Hi,
after I've updated to the latest version of MaterialIUI I receive the following error during compilation

```
Module not found: Error: Can't resolve '@material-ui/core/styles/MuiThemeProvider'
```

Turns out that they've disabled 3 level deep imports and encourage to move to ThemeProvider instead of MuiThemeProvider, see https://github.com/mui-org/material-ui/issues/17900 for details. 
I've fixed it with my own import statement, but I guess it doesn't make sense to contribute the change directly since everything is generated automatically ?

**Comment by cmeeren** [comment]
Thanks, I'll fix it in the next update!

**Comment by cmeeren** [comment]
Will be released in 0.8.0 shortly.
